.babelrc

```diff
@@ -5,15 +5,20 @@
   "env": {
     "cjs": {
       "presets": [
-        ["es2015", {"loose": true}]
+        ["latest", {"es2015": {"loose": true}}]
```

Comment:
I guess the latest preset is needed for the async function? Can we just import the async await plugin alone?